README.md

```diff
@@ -5,7 +5,7 @@
 
 The goal is to implement a simple Go style ev3dev API and helpers for common tasks.
 
-github.com/ev3go/ev3dev depends on an 18-ev3dev kernel or better (http://www.ev3dev.org/news/2017/01/25/kernel-release-cycle-18/).
+github.com/ev3go/ev3dev depends on ev3dev stretch and has been tested on kernel 4.9.50-ev3dev-1.6.0. For jessie support see the [ev3dev-jessie branch](https://github.com/ev3go/ev3dev/tree/ev3dev-jessie).
```

Comment:
The current kernel is 4.14.?-ev3dev-2.2.2, so this testing is a bit behind.

Reply:
I'll see about doing some hardware testing soon (the number here is out of date and I have tested more recently, but I don't recall versions).